cmd/chartsvc/handler_test.go

```diff
@@ -238,8 +239,11 @@ func Test_listCharts(t *testing.T) {
 			m.AssertExpectations(t)
 			assert.Equal(t, http.StatusOK, w.Code)
 
+			bb, _ := ioutil.ReadAll(w.Body)
+			assert.NotEqual(t, string(bb), `{"data":null}`, "chart list shouldn't be null")
```

Comment:
Is this test needed? Wouldn't be enough to check that it is an array of length len(tt.charts) as done below?

I am a confused though that the tests were previously passing. Is it because the assert.Len was comparing len(null) with 0?

Instead of blacklisting that case, we could check that it is an array before we check its length by for example using
https://godoc.org/github.com/stretchr/testify/assert#IsType

Reply:
Indeed, found a better way to avoid that conversion (using a pointer).